xml/admin_ds_backup.xml

```diff
@@ -140,5 +142,92 @@
     </para>
    </listitem>
   </itemizedlist>
+ </sect2>
+ </sect1>
+ <sect1 xml:id="restore-ceph">
+  <title>Restoring a Ceph Node</title>
+  <para>
+   The basic procedure for recovering a node from backup is to reinstall the
+   node, replace its configuration files, and then re-orchestrate the cluster
+   so that the replacement node is re-added. 
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     Re-install the node. (See <link
+     xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
```

Comment:
This can be an `<xref linkend:"ceph-install-stack"/>`

Reply:
I did not know it was possible to link directly to a different book. Indeed on a previous project I was told it wasn't. 

Oh well, if it works, great, it *is* much neater.

Reply:
Within the same set you can. So all the SES books can link to each other and so on and so forth. It's dependent on the hierarchical daps structure :) 
But you couldn't link SES -> SLE for example.